tests/js_tests/spec/BigBed.spec.js

```diff
@@ -0,0 +1,54 @@
+require([
+            'dojo/_base/array',
+            'JBrowse/Browser',
+            'JBrowse/Store/SeqFeature/BigBed',
+            'JBrowse/Model/XHRBlob'
+        ], function(
+            array,
+            Browser,
+            BigBed,
+            XHRBlob
+        ) {
+
+    var errorFunc = function(e) { console.error(e); };
+
+    describe( 'BigBed with volvox eden genes', function() {
+        var browser = new Browser({ unitTestMode: true });
+        var b = new BigBed({
+            browser: browser,
+            blob: new XHRBlob('../../sample_data/raw/volvox/volvox.bb')
+        });
+
+        it('constructs', function(){ expect(b).toBeTruthy(); });
+
+        (function() {
+            var features = [];
+            b.getFeatures(
+                { ref: 'ctgA', start: 1, end: 50000 },
+                function(f) { features.push(f); },
+                function() { features.done = true },
+                function(e) { console.error(e.stack||''+e); }
+            );
+
+            waitsFor( function() { return features.done; } );
+            runs( function() {
+                      //console.log( features );
+                      expect( features.length ).toEqual( 4 );
+                      var edenIndex;
+                      array.some( features, function(f,i) {
+                                      if( f.get('name') == 'EDEN.1' ) {
+                                          edenIndex = i;
+                                          return true;
+                                      }
+                                      return false;
+                                  });
+                      console.log(edenIndex);
+
+                      expect( edenIndex ).toBe( 0 );
+                      expect( features[edenIndex].get('subfeatures').length ).toEqual( 8 );
+                  });
+        }).call();
+
```

Comment:
the expect() calls above should be in an `it()` call, are you sure they are actually running?

have you tested this with larger bigbed files, like human ones?

Reply:
will be awesome to finally get bigbed support!

Reply:
I think expect() is the proper way to run assertions inside of the larger it() calls

Reply:
The main things I tested the bigbed code on were ENCODE BigBed files (simple box features) and I manually converted gencode GTF files into BigBed (gene features e.g. mRNA with exons and the thickStart/thickEnd attribute representing boundaries of CDS)

I could make those files available somewhere if there's interest but being human data they are fairly large